lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/region_event_card.dart

```diff
@@ -19,84 +20,112 @@ class RegionEventCard extends StatelessWidget {
-                      width: MediaQuery.of(context).size.width - 32,
-                      placeholder: (_, __) => const SizedBox.shrink(),
-                      errorWidget: (_, __, ___) => const SizedBox.shrink(),
-                    ),
-                  ),
-                  const SizedBox(height: 12.0),
-                  Row(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    children: [
-                      Text(
-                        event.eventName,
-                        style: Theme.of(context).textTheme.buttonLowEmphasis,
-                        overflow: TextOverflow.ellipsis,
+              child: Container(
+                decoration: state.isEventPassEndDate
+                    ? BoxDecoration(
+                        borderRadius: BorderRadius.circular(16.0),
+                        border: Border.all(color: AppColors.grey1),
+                      )
+                    : BoxDecoration(
+                        borderRadius: BorderRadius.circular(16.0),
+                        border: Border.all(color: AppColors.green3),
                       ),
-                      Text(DateFormat.yMMMMd().format(DateTime.parse(event.createdTime.toDate().toString()))),
-                    ],
-                  ),
-                  const SizedBox(height: 8.0),
-                  Row(
+                child: Padding(
+                  padding: const EdgeInsets.all(4.0),
+                  child: Column(
                     children: [
-                      BlocBuilder<RegionEventCardBloc, RegionEventCardState>(
-                        builder: (_, state) {
-                          return state.profiles.isNotEmpty
-                              ? StackedAvatars(state.profiles, 20)
-                              : const SizedBox.shrink();
-                        },
+                      Stack(
+                        children: [
+                          ClipRRect(
+                            borderRadius: BorderRadius.circular(10.0),
+                            child: CachedNetworkImage(
+                              imageUrl: event.eventImage,
+                              fit: BoxFit.cover,
+                              height: MediaQuery.of(context).size.width - 250,
+                              width: MediaQuery.of(context).size.width - 32,
+                              placeholder: (_, __) => const SizedBox.shrink(),
+                              errorWidget: (_, __, ___) => const SizedBox.shrink(),
+                            ),
+                          ),
+                          if (state.isEventPassEndDate)
+                            Positioned(
+                              bottom: 10,
+                              left: 0,
+                              child: CustomPaint(
+                                size: const Size(100, 40),
+                                painter: const CategoryLabel(color: Colors.black),
```

Comment:
should we make this a light red? 
Red usually indicated error/past etc

Reply:
new UI

![image](https://user-images.githubusercontent.com/51983923/166342919-45f8816e-7120-47fc-b107-aa4752d67f7c.png)